.changeset/nasty-shirts-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': minor
```

Comment:
This can be just patch level.

Reply:
I would consider this a breaking change as this replaces the default from title to entitiy name. 

Tough this is 0. release, so I have used minor.  

Is my understanding correct? 
Should I still change the semver level?

Reply:
I'd say patch is good, yeah. But happy to approve and leave to @backstage/discoverability-maintainers (@emmaindal maybe?) to give it a final go